src/python/WMCore/JobSplitting/EventAwareLumiByWork.py

```diff
@@ -64,7 +66,7 @@ def algorithm(self, *args, **kwargs):
         # TODO: split across sites two different jobs will read the same lumi
 
         avgEventsPerJob = int(kwargs.get('events_per_job', 5000))
-        lumiEventLimit = int(kwargs.get('max_events_per_lumi', 20000))
+        jobTimeLimit = int(kwargs.get('job_time_limit', self.defaultJobTimeLimit))
```

Comment:
We are not using this plug in now but I guess no harm to change it.

Reply:
Yes, I want to start evaluating this algo. So I think it's a good idea to keep them consistent ;)